templates/ep19/bs/header/_nav.html

```diff
@@ -0,0 +1,55 @@
+{% load cms_tags %}
+<nav class="navbar navbar-expand-lg">
+    <div class='container'>
+        <a class="navbar-brand" href="/" >EuroPython 2019</a>
+        <button class="navbar-toggler collapsed" type="button" data-toggle="collapse" data-target="#navbarsExample05" aria-controls="navbarsExample05" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+
+        <div class="navbar-collapse collapse" id="navbarsExample05" style="">
+            <ul class="navbar-nav mr-auto">
+                <li class="nav-item">
+                    <a class="nav-link" href="#">News</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="{% page_url 'program' %}">Program</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="{% page_url "speakers" %}">Speakers</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="{% page_url 'location' %}">Location</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="{% page_url 'sponsor' %}">Sponsor</a>
```

Comment:
Right now, the make dev data command does not define this route, accessing the home page throws an exception:
```
cms.models.pagemodel.DoesNotExist: A template tag couldn't find the page with lookup arguments `{'reverse_id': 'sponsor'}
`. The URL of the request was: http://example.com/
[06/Mar/2019 13:48:37] "GET / HTTP/1.1" 500 244597
```

Reply:
It does now, I fixed it -- basically to make it more like current setup on ep2018, -- but then I noticed i fixed it on separate branch, so porting that back to this PR. :)